lib/main.js

```diff
@@ -1,37 +1,38 @@
-"use strict";
+'use strict'
```

Comment:
[`quotes`](http://eslint.org/docs/rules/quotes.html) would be a good rule to to add to `.eslintrc`